concent_api/core/views.py

```diff
@@ -1443,6 +1443,114 @@ def store_message(
     return stored_message
 
 
+def update_concent_states(
+    client_public_key: str,
+):
+    active_subtask_states = [
```

Comment:
This list will be added as constant to Subtask model but it is ok for now, just a reminder to change it later.